validator/client/validator.go

```diff
@@ -364,11 +360,11 @@ func (v *validator) checkAndLogValidatorStatus(statuses []*validatorStatus, acti
 	var validatorActivated bool
 	for _, status := range statuses {
 		fields := logrus.Fields{
-			"pubKey": fmt.Sprintf("%#x", bytesutil.Trunc(status.publicKey)),
+			"pubkey": fmt.Sprintf("%#x", bytesutil.Trunc(status.publicKey)),
 			"status": status.status.Status.String(),
 		}
 		if status.index != nonexistentIndex {
-			fields["index"] = status.index
+			fields["validatorIndex"] = status.index
```

Comment:
We have `validatorIndex` in other places. I prefer this way rather than `index` because it might be unclear what kind of index it is (maybe not in this exact place, but somewhere else).